rustls/src/crypto/mod.rs

```diff
@@ -124,7 +124,7 @@ pub use crate::suites::CipherSuiteCommon;
 ///
 /// ```
 /// # #[cfg(feature = "aws_lc_rs")] {
-/// # use std::sync::Arc;
```

Comment:
I think we should stick to using `std::sync::Arc` for non-unit tests, and lose `rustls::internal::alias::atomic_sync::Arc`. Generally `internal` is used for tests to reuse code they otherwise absolutely cannot.

Reply:
@ctz I think that sounds reasonable, will try to fix this sometime later today.